### ISSUE TITLE
Bitwarden on OpenSuse fails to decode access token when changing entries... 

### ISSUE THREAD

**Issue by piedrok** [bitwarden/clients#12364]
### Steps To Reproduce

1. Go to 'bitwarden (flatpak)'
2. Click on 'any entry and try to change some information....'
3. Scroll down to 'Save changes'
4. Click on 'Save changes'

This always results in a message: 

"Failed to decode access token" 

... and the entry (including changes cannot be saved. 

This is not a duplicate to the bug in which you cannot login the flatpak app!  

The bitwarden desktop application opens the vault perfectly. Entreis can be searched and information like passwords can be copied and pasted. 

But any change or new entry results in a "Failed to decode access token". 

So basically it's completely unusable! 

Please fix this.   

### Expected Result

should save the entry and any changes

### Actual Result

does not save the entry nor any changes

### Operating System

Linux

### Operating System Version

openSUSE Tumbleweed 20241210

### Installation method

Other

### Build Version

Version 2024.11.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by piedrok** [comment]
Also: 

Restarting the app, the OS, the PC or reinstalling doesn't solved it.

**Comment by Krychaz** [comment]
Hi,

Running latest SUSE Tumbleweed I was not able to reproduce this issue. 

We use GitHub issues as a place to track bugs and other development related issues. If your issue persists, please write us back using our “Contact support” form located on our Help Center (https://bitwarden.com/help/).

You can include a link to this issue in the message content.

Alternatively, you can also search for an answer in our help documentation or get help from other Bitwarden users on our community forums (https://community.bitwarden.com/c/support/). 

The issue here will be closed.

**Comment by piedrok** [comment]
Thanks... now since today the issue disappeared - it seems to be a purely sporadic behaviour. Though googling gives me lots of messages about failing access tokens... 

Anyway, for me bitwarden desktop app works again! 

Cheers!
